dll/win32/setupapi/interface.c

```diff
@@ -374,37 +374,7 @@ InstallOneInterface(
         return FALSE;
```

Comment:
Note: the last parameter of the SetupDiCreateDeviceInterfaceRegKeyW call above should be `NULL` (not numerical "0").